server/src/test/java/org/elasticsearch/action/admin/indices/shrink/TransportResizeActionTest.java

```diff
@@ -113,84 +122,18 @@ public void testErrorCondition() {
             routingTable.index("source").shardsWithState(ShardRoutingState.INITIALIZING)).routingTable();
         clusterState = ClusterState.builder(clusterState).routingTable(routingTable).build();
 
-        TransportResizeAction.prepareCreateIndexRequest(new ResizeRequest("target", "source"), clusterState,
-            (i) -> new DocsStats(between(1, 1000), between(1, 1000), between(0, 10000)), "source", "target");
-    }
-
-    @Test
-    public void testPassNumRoutingShards() {
```

Comment:
The request is no longer able to pass along `number_of_routing_shards` (and didn't do it before), so I removed the tests.